parser_test.go

```diff
@@ -1298,6 +1298,7 @@ func (s *testParserSuite) TestBuiltin(c *C) {
 		{`SELECT tidb_is_ddl_owner();`, true, "SELECT TIDB_IS_DDL_OWNER()"},
 		{`SELECT tidb_decode_plan();`, true, "SELECT TIDB_DECODE_PLAN()"},
 		{`SELECT tidb_decode_key('abc');`, true, "SELECT TIDB_DECODE_KEY('abc')"},
+		{`SELECT get_mvcc_info('hex', '0xabc');`, true, "SELECT GET_MVCC_INFO('hex', '0xabc')"},
```

Comment:
isn't it more natural to write `get_mvcc_info('hex', 0xabc)` 🤔

Reply:
@kennytm This value is a hex decoded key, not integer value, I think use `'7480000000000000FF2E5F698000000000FF0000010380000000FF05F5E10003000000FF0000000000000000F7'` is better.

Reply:
ok. then the input should be `'0abc'` 🙃.

also note that in MySQL `0xabc` is not an integer, but a binary blob.

```
mysql> select 0x416464206275696C74696E2066756E6374696F6E2070726F746F7479706520606765745F6D7663635F696E666F6020696E20706172736572;
+--------------------------------------------------------------------------------------------------------------------+
| 0x416464206275696C74696E2066756E6374696F6E2070726F746F7479706520606765745F6D7663635F696E666F6020696E20706172736572 |
+--------------------------------------------------------------------------------------------------------------------+
| Add builtin function prototype `get_mvcc_info` in parser                                                           |
+--------------------------------------------------------------------------------------------------------------------+
1 row in set (0.00 sec)

mysql> select x'416464206275696C74696E2066756E6374696F6E2070726F746F7479706520606765745F6D7663635F696E666F6020696E20706172736572';
+---------------------------------------------------------------------------------------------------------------------+
| x'416464206275696C74696E2066756E6374696F6E2070726F746F7479706520606765745F6D7663635F696E666F6020696E20706172736572' |
+---------------------------------------------------------------------------------------------------------------------+
| Add builtin function prototype `get_mvcc_info` in parser                                                            |
+---------------------------------------------------------------------------------------------------------------------+
1 row in set (0.00 sec)
```